docs/docs/explanations/advanced-provider-configuration.md

```diff
@@ -0,0 +1,205 @@
+azure:
+  region: Central US
+  kubernetes_version: 1.19.11
+  node_groups:
+    general:
+      instance: Standard_D4_v3
+      min_nodes: 1
+      max_nodes: 1
+    user:
+      instance: Standard_D2_v2
+      min_nodes: 0
+      max_nodes: 5
+    worker:
+      instance: Standard_D2_v2
+      min_nodes: 0
+      max_nodes: 5
+  storage_account_postfix: t65ft6q5
+```
+
+</TabItem>
+
+<TabItem value="do" label="DigitalOcean">
+
+DigitalOcean has a restriction with autoscaling in that the minimum nodes allowed (`min_nodes` = 1) is one but is by far the least expensive provider even accounting for `spot/pre-emptible` instances.
+In addition, Digital Ocean doesn't have accelerator/gpu support.
+
+Digital Ocean is a good choice for trying out Nebari, but we recommend selecting a different provider for your production Nebari deployment.
+
+To see available instance types refer to [Digital Ocean Instance Types](https://www.digitalocean.com/docs/droplets/).
+Additionally the Digital Ocean cli `doctl` has [support for listing droplets](https://www.digitalocean.com/docs/apis-clis/doctl/reference/compute/droplet/list/).
+
+```yaml
+digital_ocean:
+  region: nyc3
+  kubernetes_version: "1.21.10-do.0"
+  node_groups:
+    general:
+      instance: "g-4vcpu-16gb"
+      min_nodes: 1
+      max_nodes: 1
+    user:
+      instance: "g-2vcpu-8gb"
+      min_nodes: 1
+      max_nodes: 5
+    worker:
+      instance: "g-2vcpu-8gb"
+      min_nodes: 1
+      max_nodes: 5
+```
+
+</TabItem>
+
+<TabItem value="existing" label="Existing Kubernetes clusters">
+
+Originally designed for Nebari deployments on a "local" minikube cluster, this feature has now expanded to allow users to deploy Nebari to any existing kubernetes cluster.
+The default options for an `existing` deployment are still set to deploy to a minikube cluster.
+
+<!-- TODO: Uncomment and add link when available -->
+<!-- If you wish to deploy to an existing kubernetes cluster on one of the cloud providers, please refer to a more detailed walkthrough found in the [Deploy Nebari to an Existing Kubernetes Cluster]. -->
+
+Deploying to a local existing kubernetes cluster has different options than the cloud providers. `kube_context` is an optional key that can be used to deploy to a non-default context.
+The default node selectors will allow pods to be scheduled anywhere.
+This can be adjusted to schedule pods on different labeled nodes.
+Allowing for similar functionality to node groups in the cloud.
+
+```yaml
+### Provider configuration ###
+existing:
+  kube_context: minikube
+  node_selectors:
+    general:
+      key: kubernetes.io/os
+      value: linux
+    user:
+      key: kubernetes.io/os
+      value: linux
+    worker:
+      key: kubernetes.io/os
+      value: linux
+```
+
+</TabItem>
+
+<TabItem value="local" label="Local (testing)">
+
+Local deployment is intended for Nebari deployments on a "local" cluster created and management by Kind.
+It is great for experimentation and development.
+
```

Comment:
Insert some comment here about Macs - @iameskild can you do this?

Reply:
I'll add a note saying this is only supported on linux right now, like in: http://localhost:3000/docs/how-tos/nebari-local#compatibility

Let me know if you'd like to add something more. :)